packages/directives/vue-visible-on-scroll/src/index.js

```diff
@@ -0,0 +1,70 @@
+export default {
+  mounted(el, binding) {
+    element = el;
+    routeWithHash = binding.value; // expects $route.hash to be passed
+
+    if (process.browser && onDesktop()) {
+      enableVisibleOnScroll();
+    }
+
+    window.addEventListener("resize", handleResize);
+  },
+
+  // TODO remove even listeners on unmounted? Current use in header, is always mounted
```

Comment:
Yep, let's do this now, lest we forget when this is used elsewhere later on.

Reply:
Done.